std_msgs/Header header
int64 ctl_modeFromCommand     # ctl_mode reading from 0-4

